runtime/bindings/python/tests/test_inference_engine/test_infer_request.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import os
+import pytest
+
+from openvino import Core, Blob, TensorDesc, StatusCode
+
+
+def image_path():
+    path_to_repo = os.environ["DATA_PATH"]
+    path_to_img = os.path.join(path_to_repo, "validation_set", "224x224", "dog.bmp")
+    return path_to_img
+
+
+def model_path(is_myriad=False):
+    path_to_repo = os.environ["MODELS_PATH"]
+    if not is_myriad:
+        test_xml = os.path.join(path_to_repo, "models", "test_model", "test_model_fp32.xml")
+        test_bin = os.path.join(path_to_repo, "models", "test_model", "test_model_fp32.bin")
+    else:
+        test_xml = os.path.join(path_to_repo, "models", "test_model", "test_model_fp16.xml")
+        test_bin = os.path.join(path_to_repo, "models", "test_model", "test_model_fp16.bin")
+    return (test_xml, test_bin)
```

Comment:
can we generate IRs at the beginning of the tests and remove upon the finish? It's not convenient to work with env variables and extra repositories

Reply:
discussed offline. Created ticket: 67090